### PR TITLE
[JEP-200] [JENKINS-49024] Possible fix for serialization error

### PR DESCRIPTION
**Pull request by jglick**
[JENKINS-49024](https://issues.jenkins-ci.org/browse/JENKINS-49024)

@reviewbybees

### CODE REVIEW

**Comment by jglick** [comment]
I do not think there should be any issues with loading old `config.xml`, as a `static` field should not even be considered during deserialization.

Anyway, @reviewbybees done and we will see if a maintainer appears.

File: pom.xml [hunk 1]
```diff
@@ -78,37 +84,6 @@
...
-                <configuration>
-                    <links>
-                        <link>https://hudson.dev.java.net/nonav/javadoc/</link>
-                        <link>https://stapler.dev.java.net/nonav/javadoc/</link>
-                    </links>
-                </configuration>
-            </plugin>
-
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
```
Comment (oleg-nenashev) [inline pom.xml]: Default Parent POM does not offer Cobertura (it offers jacoco instead). But I doubt anybody really cares about the code coverage in the current state

**Comment by jglick** [comment]
@reviewbybees done

**Comment by oleg-nenashev** [comment]
@blalor Are you fine if somebody makes a plugin release and then marks the plugin for adoption? I doubt you're still interested in this plugin after 6 years, but let us know just in case

**Comment by blalor** [comment]
Yikes.  More like 9 years ago!  Man, I barely remember _writing_ this code.  Do with it what you wish!

**Pull request merged**